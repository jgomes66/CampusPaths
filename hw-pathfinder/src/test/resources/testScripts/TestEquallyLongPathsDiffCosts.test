# Simple test to ensure that the least costing
# path is returned when there are multiple shortest paths

CreateGraph simpleGraph

AddNode simpleGraph v1
AddNode simpleGraph v2
AddNode simpleGraph v3
AddNode simpleGraph v4

AddEdge simpleGraph v1 v2 1
AddEdge simpleGraph v2 v4 1
AddEdge simpleGraph v1 v3 1
AddEdge simpleGraph v3 v4 2

FindPath simpleGraph v1 v4
