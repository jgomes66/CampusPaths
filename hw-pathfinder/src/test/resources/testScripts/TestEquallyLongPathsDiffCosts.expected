# Simple test to ensure that the least costing
# path is returned when there are multiple shortest paths

created graph simpleGraph

added node v1 to simpleGraph
added node v2 to simpleGraph
added node v3 to simpleGraph
added node v4 to simpleGraph

added edge 1.000 from v1 to v2 in simpleGraph
added edge 1.000 from v2 to v4 in simpleGraph
added edge 1.000 from v1 to v3 in simpleGraph
added edge 2.000 from v3 to v4 in simpleGraph

path from v1 to v4:
v1 to v2 with weight 1.000
v2 to v4 with weight 1.000
total cost: 2.000

